Features/links.feature

```diff
@@ -0,0 +1,77 @@
+# language: nl
+
+Voor tijdreisvragen endpoints:
+ opbouw self link:
+ Er wordt exact overgenomen wat er door een client is opgegeven.
```

Comment:
Dit geldt niet als het de vraag is opgenomen als een embedded resource; dat mag denk ik er nog bij als verduidelijking

Reply:
@kad-wettut gewenst gedrag voor levenscyclusendpoints is gewijzigd naar gewenst gedrag voor endpoints die collecties leveren.
En boven de scenario's gezet om dit duidelijker in beeld te hebben.